cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/CFPoint.java

```diff
@@ -222,4 +249,16 @@ public CFPoint(double x, double y, CFPoint prev, CFPoint next, Array data) {
 		this.y = y;
 		this.data = data;
 	}
+	
+	/**
+	 * Constructs a new empty point at (0,0) with no connections.
+	 * 
+	 */
+	public CFPoint() {

```

Comment:
Have you thought about calling the other CFPoint constructor here?

Reply:
Good POINT, could probably get away with constructor composition here.

Reply:
😆